docs/user-guide/how_to_use_hypernode_auto_discovery.md

```diff
@@ -0,0 +1,94 @@
+# Usage Document
+
+## Introduction
+
+This design document describes the design and implementation of the HyperNode network topology discovery feature in Volcano. This feature automatically discovers the network topology structure within the cluster and creates and maintains HyperNode custom resources (CRs) based on the discovered topology information. Consequently, the Volcano Scheduler will leverage these HyperNode CRs for scheduling decisions, eliminating the need for users to manually maintain HyperNode information.
+
+## Prerequisites
+
+Please follow [this guide](https://volcano.sh/en/docs/v1-11-0/network_topology_aware_scheduling/#installing-volcano) to install Volcano with Network Topology Aware Scheduling feature enabled.
+
+## Configuration
+
+The HyperNode network topology discovery feature is configured via a ConfigMap. The ConfigMap contains the configuration for the discovery sources, such as UFM, RoCE, and label, you can modify the configuration according to your own cluster environments.
+Please note that you should replace with your Volcano namespace if Volcano is not installed in the default namespace.
+
+### Example ConfigMap
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: volcano-controller-configmap
+  namespace: volcano-system # Replace with your Volcano namespace if Volcano is not installed in the default namespace.
+data:
+  volcano-controller.conf: |
+    discoveryConfig:
+      - source: ufm
+        enabled: true
+        interval: 10m
+        config:
+          endpoint: https://ufm-server:8080
+          username: admin
+          password: password
+          insecureSkipVerify: true
+      - source: roce
+        enabled: false
+        interval: 15m
+        config:
+          endpoint: https://roce-server:9090
```

Comment:
`config.endpoint` makes sense for UFM since it provides a centralized interface, but RoCE doesn’t have centralized management. What is this field intended to connect to in the case of RoCE?

Reply:
That's a good catch, currentlt roce type is not supported, and we have no roce environment to test this feature. Do you have any idea what fields should be filled in roce environment？

Reply:
As we discussed earlier, for RoCE we can use LLDP to collect neighbor information from each node and switch, then reconstruct the full topology. On the node side, tools like https://github.com/lldpd/lldpd or other open-source alternatives can be used. However, for switches, LLDP support and access methods can vary significantly between vendors. This makes it challenging to maintain compatibility across all switch types and may require contributions from the vendors themselves.

Reply:
Ok，I see. This interface is extensible enough to cover RoCE in the future, if we need a daemonset we can add it in RoCE scenario.